homeassistant/components/sensor/netatmo.py

```diff
@@ -112,9 +117,11 @@ def __init__(self, netatmo_data, module_name, sensor_type):
         self._device_class = SENSOR_TYPES[self.type][3]
         self._icon = SENSOR_TYPES[self.type][2]
         self._unit_of_measurement = SENSOR_TYPES[self.type][1]
-        module_id = self.netatmo_data.\
+        self._module_type = self.netatmo_data. \
+            station_data.moduleByName(module=module_name)['type']
```

Comment:
Why you use this variable?

Reply:
It is needed in the update function to distinguish the module type to calculate the correct battery level.